etc/oods/config/ncsa/cc_oods.yaml

```diff
@@ -36,7 +36,8 @@ cacheCleaner:
         - /data/lsstdata/NTS/comcam/oods/butler/repo/raw
         - /data/lsstdata/NTS/comcam/oods/butler/badfiles
         - /data/lsstdata/NTS/comcam/oods/butler/staging
-        - /data/lsstdata/NTS/comcam/oods/gen3butler/repo/raw
+        - /data/lsstdata/NTS/comcam/oods/gen3butler/raw
+        - /data/lsstdata/NTS/comcam/oods/gen3butler/repo/LSSTComCam/raw
```

Comment:
Similarly truncate the `raw` here and in the other files below.

Reply:
See above.